robolectric/src/test/java/org/robolectric/RobolectricTestRunnerSelfTest.java

```diff
@@ -107,6 +107,11 @@ public void testVersionConfiguration() {
         .isEqualTo("4.4_r1");
   }
 
+
+  @Test public void assertThatz() throws Exception {
```

Comment:
Fixed the test name. I came across a classloader conflict because `org.junit.Assert` loads some hamcrest things.